packages/console/src/components/Switch/index.module.scss

```diff
@@ -0,0 +1,62 @@
+@use '@/scss/underscore' as _;
+
+.switch {
+  position: relative;
+  display: inline-block;
+  width: 40px;
+  height: 24px;
+
+  input {
+    opacity: 0%;
+    width: 0;
+    height: 0;
+  }
+
+  .slider {
+    position: absolute;
+    cursor: pointer;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    background-color: var(--color-neutral-90);
+    transition: 0.4s;
+    transition: 0.4s;
+    border-radius: 24px;
+
+    &::before {
+      position: absolute;
+      content: '';
+      height: 20px;
+      width: 20px;
+      left: 2px;
+      bottom: 2px;
+      background-color: var(--color-on-primary);
+      transition: 0.4s;
+      border-radius: 50%;
+      box-shadow: 0 3px 7px var(--color-shadow-12);
```

Comment:
--shadow-light-s2

Reply:
It doesn't match, `--shadow-ligth-s2`'s shadow is too large, the shadow here in switch is a bit different from other shadows.

Reply:
We'd better check it out with designers

Reply:
updated to hard-code value, since is a very special case.

Reply:
ok